client/src/main.jsx

```diff
@@ -1,19 +1,33 @@
 "use client";
 import React from "react";
 import ReactDOM from "react-dom/client";
+import { Auth0Provider } from "@auth0/auth0-react";
 import { ErrorBoundary } from "react-error-boundary";
 import App from "./App.jsx";
 import { store } from "./app/store.jsx";
 import { Provider } from "react-redux";
 import GlobalStyles from "./styles/GlobalStyles.jsx";
 
+const domain = import.meta.env.VITE_AUTH0_DOMAIN;
```

Comment:
if you add config/env virables, you need to say it and update the config gen script